library/src/app/components/trade/trade.component.html

```diff
@@ -0,0 +1,102 @@
+<div
+  [ngClass]="{
+    'cybrid-dark-theme': (configService.config$ | async)?.theme === 'DARK'
+  }"
+>
+  <ng-container *ngIf="(isRecoverable$ | async) !== false; else error">
+    <ng-container *ngIf="(isLoading$ | async) !== true">
+      <ng-container [formGroup]="quoteGroup" *ngIf="quote$ | async as quote">
+        <div fxLayout="column" fxLayoutAlign="center">
+          <mat-form-field appearance="outline">
+            <mat-label>
+              <span>{{ 'trade.amount' | translate }}</span>
+              <span class="code">
+                {{
+                  input === 'fiat' ? quote.asset.code : quote.counterAsset.code
```

Comment:
How will this change if there is a swap being performed, i.e., crypto for crypto?

Reply:
Right now you can't get into a crypto to crypto state. Clicking from the price list will always load crypto-fiat. Maybe we need to rethink the flow a bit?

Reply:
Just pinged Michael and it sounds like we can't do crypto-crypto right now. This addition would mean adding a new input field and so the code would have to change a bit.

Reply:
Chatted with Faramarz about this as well. Typical industry UI for this would have a tab for swap and a different template with a `asset` to `counter-asset` set of selects.